service/storage_service.cc

```diff
@@ -434,7 +434,7 @@ future<> storage_service::topology_transition(storage_proxy& proxy, cdc::generat
         }
     } catch (replica::no_such_column_family& e) {
         slogger.error("Error while applying topology mutations from {}: {}", from, e);
-        throw std::runtime_error(fmt::format("Error while applying topology mutations: {}", e));
+        throw std::runtime_error(format("Error while applying topology mutations: {}", e));
```

Comment:
Please use seastar::format or ::format (same thing). Unqualified format will also match std::format using argument-dependent format, and so will be ambiguous, once we switch to libstdc++ 13.

Reply:
below as well.